src/puppetlabs/puppetdb/cli/benchmark.clj

```diff
@@ -245,10 +265,15 @@
   [hosts num-msgs]
   (log/infof "Sending %s messages for %s hosts, will exit upon completion"
              num-msgs (count hosts))
-  (loop [mutated-hosts hosts
-         msgs-to-send num-msgs]
-    (when-not (zero? msgs-to-send)
-      (recur (mapv update-host mutated-hosts) (dec msgs-to-send)))))
+  (let [stamp (atom (time/minus (time/now) (time/minutes (* 30 num-msgs))))]
```

Comment:
Why do we need an atom here? Seems like it could be another loop var pretty easily...